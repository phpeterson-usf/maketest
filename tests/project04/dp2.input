-h $(repo_dir)/dp2.hex $(TESTS_DIR)/dp2.s
