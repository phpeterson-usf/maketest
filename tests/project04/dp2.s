main:
    mov r0, r1
    bx lr
