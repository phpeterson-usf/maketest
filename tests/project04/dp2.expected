v2.0 raw
e1a00001
e12fff1e
